Draft defintion of steering committee (#224)
FamilySearch/GEDCOM.io: _pages/community.md

If you are associated with an application and want to add it to this listing or change its existing entry in this listing, send an email to <GEDCOM@FamilySearch.org>.

> ## Steering Committee

Reviewer: ```suggestion
## GEDCOM Steering Committee
```
Suggest using the section title matching what we actually call the committee so a web search on the full name would find this section.

The GEDCOM standard is overseen by a steering committee who meet weekly to discuss issues and pull requests on the GEDCOM  repositories.

> Most of the work of developing and refining the standard are done between those meetings,

Author: ```suggestion
Most of the work of developing and refining the standard is done between those meetings,
```

either by individuals or project teams.

> That work consists of identifying issues, proposing alternatives, and writing those up as pull requests.

Author: ```suggestion
That work consists of identifying issues, proposing changes, and converting those change proposals into pull requests.
```

> The weekly steering committee meetings primarily discuss the issues and pull requests resulting from that project team and individual work.

Reviewer: ```suggestion
The weekly steering committee meetings primarily discuss the issues and pull requests resulting from that project team or individual work.
```
grammar suggestion

> When the current steering committee notices that someone not on the committee is making consistent helpful contributions spanning several areas of the specification over an extended time,

Reviewer: ```suggestion
When the current GEDCOM Steering Committee notices that someone not on the committee is making consistent helpful contributions spanning several areas of the specification over an extended time,
```

they invite that person to join the steering committee.

> ## Project Teams

Reviewer: ```suggestion
### Project Teams

The GEDCOM Steering Committee also organizes Project Teams for more complex projects.
```

If you are associated with an application and want to add it to this listing or change its existing entry in this listing, send an email to <GEDCOM@FamilySearch.org>.

## GEDCOM Steering Committee

> The GEDCOM standard is overseen by a steering committee who meet weekly to discuss issues and pull requests on the GEDCOM  repositories.

Reviewer: ```suggestion
The GEDCOM standard is overseen by a steering committee who meet weekly to discuss issues and pull requests on the GEDCOM repositories.
```
nit

Most of the work of developing and refining the standard is done between those meetings,
either by individuals or project teams.
That work consists of identifying issues, proposing changes, and converting those change proposals into pull requests.
The weekly steering committee meetings primarily discuss the issues and pull requests resulting from that project team or individual work.

> When the current GEDCOM Steering Committee notices that someone not on the committee is making consistent helpful contributions spanning several areas of the specification over an extended time,

Author: ```suggestion
The GEDCOM Steering Committee was initially formed by FamilySearch from volunteers representing several companies and groups.
When the current GEDCOM Steering Committee notices that someone not on the committee is making consistent helpful contributions spanning several areas of the specification over an extended time,
```
